main/read.c

```diff
@@ -296,6 +296,39 @@ static boolean parseLineDirective (void)
  *   Input file I/O operations
  */
 
+#define MAX_IN_MEMORY_FILE_SIZE (1024*1024)
+
+static MIO *getMio (const char *const fileName, const char *const openMode)
+{
+	char buf[4096];
+	MIO *src, *dst;
+	size_t size = 0;
+
+	src = mio_new_file (fileName, openMode);
+	dst = mio_new_memory (NULL, 0, eRealloc, free);
```

Comment:
Yes, sorry. But is it any relevant now? The patch was completely rewritten by https://github.com/universal-ctags/ctags/pull/870/commits/c5e3c54615da16d41de82162f1cc2246a2007d5e and this isn't present in the code any more (the same applies to the second comment).